### PR TITLE
Updated registration view with unerage message

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#9372](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9372)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Updated registration for underage and message

image

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### CODE REVIEW

**Review by sslaws — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 1]
```diff
@@ -66,6 +66,9 @@ export default class RegistrationView extends Vue {
 
     private waitlistedSuccessfully = false;
 
+    private isValidAge = false;
+    private minimunAge!: number;
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:70]: minimumAge instead :-)

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 2]
```diff
@@ -74,6 +77,8 @@ export default class RegistrationView extends Vue {
             SERVICE_IDENTIFIER.BetaRequestService
         );
 
+        this.minimunAge = 16; //this.webClientConfig.minPatientAge;
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:80]: minimumAge

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 3]
```diff
@@ -486,145 +502,166 @@ export default class RegistrationView extends Vue {
...
+                                variant="primary"
+                                :class="{ disabled: !accepted }"
+                                >Register</b-button
+                            >
+                        </b-col>
+                    </b-row>
+                </b-form>
+                <div v-else>
+                    <h1>Minimun age required for registration</h1>
+                    You must be <strong>{{ minimunAge }}</strong> years of age
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:661]: minimum

File: Apps/WebClient/src/ClientApp/src/views/registration.vue [hunk 4]
```diff
@@ -486,145 +502,166 @@ export default class RegistrationView extends Vue {
...
+                                size="lg"
+                                variant="primary"
+                                :class="{ disabled: !accepted }"
+                                >Register</b-button
+                            >
+                        </b-col>
+                    </b-row>
+                </b-form>
+                <div v-else>
+                    <h1>Minimun age required for registration</h1>
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/views/registration.vue:660]: Minimum age

File: Apps/WebClient/src/ClientApp/src/services/restBetaRequestService.ts [hunk 5]
```diff
@@ -23,7 +23,7 @@ export class RestBetaRequestService implements IBetaRequestService {
         this.http = http;
     }
 
-    public getRequest(hdid: string): Promise<BetaRequest> {
+    public getRequested(hdid: string): Promise<BetaRequest> {
```
Comment (leobcgov) [inline Apps/WebClient/src/ClientApp/src/services/restBetaRequestService.ts:26]: naming issue, how about just call it "get" as it is a method of BetaRequestService :-)

File: Apps/WebClient/src/ClientApp/src/views/registration.vue (hunk 2 above)
Comment (leobcgov) [reply Apps/WebClient/src/ClientApp/src/views/registration.vue:80]: also shouldn't it be hard-coded here?

**Comment by leobcgov** [comment]
also WebClient Build has failed.

**Review by leobcgov — CHANGES_REQUESTED** [review]

File: Apps/WebClient/src/ClientApp/src/services/restBetaRequestService.ts (hunk 5 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/services/restBetaRequestService.ts:26]: I dont think get is better, although I am not sure what to name it either :)

File: Apps/WebClient/src/ClientApp/src/views/registration.vue (hunk 1 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/registration.vue]: fixed

File: Apps/WebClient/src/ClientApp/src/views/registration.vue (hunk 2 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/registration.vue]: fixed

File: Apps/WebClient/src/ClientApp/src/views/registration.vue (hunk 3 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/registration.vue]: fixed

File: Apps/WebClient/src/ClientApp/src/views/registration.vue (hunk 4 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/ClientApp/src/views/registration.vue]: fixed

**Review by leobcgov — APPROVED** [review]

**Review by sslaws — APPROVED** [review]

**Pull request merged**